buildah.go

```diff
@@ -19,7 +19,7 @@ const (
 	// identify working containers.
 	Package = "buildah"
 	// Version for the Package
-	Version       = "0.0.1"
+	Version       = "0.1"
 	containerType = Package + " 0.0.1"
```

Comment:
So are we leaving this alone? Or modifying it.  If we leave it alone, you need to add a comment explaining what it is used for?